common/xdc/event_generator.go

```diff
@@ -45,6 +45,7 @@ type (
 		GetNextVertex() []Vertex
 		ListGeneratedVertex() []Vertex
 		Reset()
+		RemoveVertexFromNextVertexCandidates(Vertex)
```

Comment:
plz add comments describing the terminologies, especially what does these means in the context of Cadence

Reply:
Updated with more detailed doc.  #2280